<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
 <class>DatasetExportWidget</class>
 <widget class="QWidget" name="DatasetExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dataset Export</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftPanelLayout">
       <item>
        <widget class="TagFilterWidget" name="dbSearchWidget" native="true">
         <layout class="QVBoxLayout" name="filterLayout" />
        </widget>
       </item>
       <item>
        <widget class="ThumbnailSelectorWidget" name="thumbnailSelector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="imageCountLabel">
         <property name="text">
          <string />
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightPanelLayout">
       <item>
        <widget class="ImagePreviewWidget" name="imagePreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="exportGroupBox">
         <property name="title">
          <string>Export Settings</string>
         </property>
         <layout class="QVBoxLayout" name="exportLayout">
          <item>
           <widget class="DirectoryPickerWidget" name="exportDirectoryPicker" native="true" />
          </item>
          <item>
           <widget class="QLabel" name="resolutionLabel">
            <property name="text">
             <string>処理済み画像解像度:</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxResolution">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>512px</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>768px</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1024px</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1536px</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="exportFormatLabel">
            <property name="text">
             <string>エクスポート形式:</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="formatSelectionLayout">
            <item>
             <widget class="QRadioButton" name="radioTxtSeparate">
              <property name="text">
               <string>TXT形式（タグ+キャプション別ファイル）</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioTxtMerged">
              <property name="text">
               <string>TXT形式（キャプション統合）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioJson">
              <property name="text">
               <string>JSON形式（metadata.json）</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="latestOnlyCheckBox">
            <property name="text">
             <string>最新のアノテーションのみ使用</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="validationResultsLabel">
            <property name="text">
             <string>検証結果:</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="validationStatsLayout">
            <item>
             <widget class="QLabel" name="totalImagesLabel">
              <property name="text">
               <string>対象画像数: --</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="validImagesLabel">
              <property name="text">
               <string>エクスポート可能: --</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="errorCountLabel">
              <property name="text">
               <string>エラー: --</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="validationDetailsText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>検証実行ボタンをクリックして詳細を確認してください。</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="actionButtonsLayout">
            <item>
             <widget class="QPushButton" name="validateButton">
              <property name="text">
               <string>検証実行</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportButton">
              <property name="text">
               <string>エクスポート開始</string>
              </property>
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="controlButtonsLayout">
            <item>
             <widget class="QPushButton" name="cancelButton">
              <property name="text">
               <string>キャンセル</string>
              </property>
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="text">
               <string>閉じる</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="exportProgressBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="text">
             <string>準備完了</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagePreviewWidget</class>
   <extends>QWidget</extends>
   <header>..widgets.image_preview</header>
   <container>1</container>
   <slots>
    <slot>load_image()</slot>
    <slot>update_model_options()</slot>
    <slot>send_vision_prompt()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ThumbnailSelectorWidget</class>
   <extends>QWidget</extends>
   <header>..widgets.thumbnail</header>
   <container>1</container>
   <slots>
    <signal>on_thumbnail_clicked()</signal>
    <signal>imageSelected()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>DirectoryPickerWidget</class>
   <extends>QWidget</extends>
   <header>..widgets.directory_picker</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagFilterWidget</class>
   <extends>QWidget</extends>
   <header>..widgets.filter</header>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>
